finish/README.md

```diff
@@ -1,10 +1,9 @@
 # Setup
-
-To use the sample application, extract the [sampleapp_bulkhead.zip] (https://github.com/OpenLiberty/iguide-bulkhead/raw/master/finish/sampleapp_bulkhead.zip) file to your local directory.
-
-Use the 'mvn install' Maven command from the directory that contains the extracted .zip files to build the project and install it in your local repository. The command creates the `target/liberty` directory that contains your Liberty server, bulkheadSampleServer, and starts the server in the background.
-
-To stop the running server, run the Maven command `mvn liberty:stop-server` from the <extract-directory> directory. To start the bulkheadSampleServer, run the Maven command `mvn liberty:start-server` in the <extract-directory> directory.
+From inside the finish directory, build and start the application in Open Liberty with the following command:
```

Comment:
We really shouldn't be storing zips in git repos, particularly since we're expecting maven as a prereq. I don't believe any of the non-interactive guides do this either